lib/log_analyzer/analyzer.rb

```diff
@@ -40,7 +40,7 @@ def report_results(build)
     end
 
     def log(msg)
-      puts msg
+      puts "LogAnalyzer: #{msg}"
```

Comment:
What about using a logger? So we can manage more options: decide where to output the logs, level of detail, etc.

Reply:
Done!